setup.py

```diff
@@ -34,6 +35,10 @@
                'Topic :: Software Development :: Quality Assurance',
                'Topic :: Software Development :: Testing']
 
+file_dir = os.path.join(base_dir, 'CHANGELOG.md')
```

Comment:
This change is nice but not necessary for Travis v3 API. Can be done in a separate PR.

Reply:
okay reverted to original file.